insights/core/dr.py

```diff
@@ -705,6 +710,8 @@ def __init__(self, *deps, **kwargs):
         tags.extend(kwargs.get("tags", []) or [])
         self.tags = set(tags)
 
+        self.timeout = kwargs.get("timeout", 120)
```

Comment:
Do you think it's necessary to define a global variable for the default timeout for datasources?

Reply:
I didn't think so I based it off of how simple_command has a default timeout set, and has the parameter option to set it manually. I didn't want to have to pass in the global variable every time a datasource is defined, because we could easily forget to add the timeout parameter when adding new ones. But if you think that logic isn't sound, I can definitely add a global variable like how your PR was setup.

Reply:
No worries, I think your implement that applying the `120` timeout to each datasource by default and adjusting it only when necessary is better.